compare_mt_assembly_to_ref.md

```diff
@@ -0,0 +1,327 @@
+# Comparing our de novo transcriptome assembly to the reference
+
+In the [transcriptome assembly lesson](https://angus.readthedocs.io/en/2018/transcriptome-assembly.html), we assembled short RNA reads into contigs. Here we will compare this to the reference transcriptome we worked with in the [mapping and variant calling](https://angus.readthedocs.io/en/2018/mapping-variant-calling.html) lesson, and get some more practice at the command line and working with blast!  
+
+---
+
+Learning objectives:
+
+	* Explore some more bash commands in a practical way (e.g. `grep`, `sed`, `cut`, `comm`, `cat`)
+	* Practice stringing together multiple commands with pipes (`|`) step-by-step
+	* Use blast to compare two transcriptomes to find sequences that are different
+	* Run a "remote" blast from the command line
+
+## Boot up a Jetstream
+[Boot an m1.medium Jetstream instance](https://angus.readthedocs.io/en/2018/jetstream/boot.html) and log in.
+
+## Software needed
+We will be using blast here, so if you've run through the [using blast at the command line](https://angus.readthedocs.io/en/2018/running-command-line-blast.html) lesson, you're already good to go. You can check that you have blast by entering:
+
+```bash
+makeblastdb -h
+```
+
+If that returns some help information, blast is already installed. If it returns a message with "command not found", then you can install blast with:
+
+```bash
+conda install -y blast
```

Comment:
there's no harm in simply running this, is there? no need to check if it's already installed... just a comment :)

Reply:
I was thinking for continuity's sake if they remembered installing blast already it'd be nice for them to see it's still there. But yea, no harm in straight install command on these wonderful setups :)